setup_cfg_fmt.py

```diff
@@ -100,10 +102,14 @@ def cb(match: Match[str]) -> str:
     return GLOB_PART.sub(cb, s)
 
 
-def _first_file(setup_cfg: str, prefix: str) -> str | None:
+def _first_file(
+    setup_cfg: str,
+    prefix: str,
+    sort_key: Callable[[str], Any] | None = None,
```

Comment:
i could not figure out how to properly type the return value of the key parameter, 
only found this typeshed when trying to look at the source through my IDE: https://github.com/python/typeshed/blob/master/stdlib/_typeshed/__init__.pyi#L70